frontend/package.json

```diff
@@ -51,6 +51,7 @@
     "react-toastify": "^10.0.4",
     "react-tooltip": "^5.26.3",
     "react-vis": "^1.12.1",
+    "ruff": "^1.5.4",
```

Comment:
@owlester12 `ruff` is a Python linter and shouldn't be in `package.json`. Can you remove this please?

cc @rm03